ibtk/include/ibtk/ibtk_utilities.h

```diff
@@ -119,6 +119,18 @@ static const bool ENABLE_TIMERS = true;
 
 namespace IBTK
 {
+#ifdef IBTK_HAVE_MAKE_UNIQUE
+#define ibtk_make_unique std::make_unique
+#else
+template <typename T, typename... Args>
+std::unique_ptr<T>
+make_unique(Args&&... args)
+{
+    return std::unique_ptr<T>(new T(std::forward<Args>(args)...));
+}
+#define ibtk_make_unique IBTK::make_unique
+#endif
```

Comment:
Same here - we should just have `IBTK::make_unique()` and we can deprecate that at some point when we move to C++14.